README.md

```diff
@@ -6,12 +6,13 @@
 [![CI](https://github.com/meggart/DiskArrays.jl/actions/workflows/ci.yml/badge.svg)](https://github.com/meggart/DiskArrays.jl/actions/workflows/ci.yml)
 [![Codecov](https://codecov.io/gh/meggart/DiskArrays.jl/branch/main/graph/badge.svg)](https://codecov.io/gh/meggart/DiskArrays.jl/tree/main)
 
-This package is an attempt to collect utilities for working with n-dimensional array-like data
-structures that do not have considerable overhead for single read operations. Most important
-examples are arrays that represent data on hard disk that are accessed through a C
-library or that are compressed in chunks. It can be inadvisable to make these arrays a subtype
-of `AbstractArray` many functions working with AbstractArrays assume fast random access into single
-values (including basic things like `getindex`, `show`, `reduce`, etc...). Currently supported features are:
+This package provides a collection of utilities for working with n-dimensional array-like data
+structures that do not have considerable overhead for single read operations. 
+Most important examples are arrays that represent data on hard disk that are accessed through a C
+library or that are compressed in chunks. 
+It can be inadvisable to make these arrays a subtype of `AbstractArray` many functions working with AbstractArrays assume fast random access into single values (including basic things like `getindex`, `show`, `reduce`, etc...). 
```

Comment:
Isn't this in contradiction to DiskArrays itself being a subtype of AbstractArray?

Reply:
Maybe it should say "direct subtype"